src/Calendar.js

```diff
@@ -175,6 +175,17 @@ class Calendar extends React.Component {
      */
     endAccessor: accessor,
 
+    /**
+     * Determine if an event is draggable. Must resolve to a Boolean
+     *
+     * ```js
+     * string | (event: Object) => Boolean
+     * ```
+     *
+     * @type {(func|string)}
+     */
+    draggableAccessor: accessor,
```

Comment:
This should go in https://github.com/dbuchet-bandc/react-big-calendar/blob/f2c9cf454b795064076b879c272192897ab2b4ec/src/addons/dragAndDrop/withDragAndDrop.js#L65 since it's not relevant to the core calendar only the addon